### ISSUE TITLE
Deferred deep linking with clipboard

### ISSUE THREAD

**Issue by agazso** [felfele/felfele#452]
**Is your feature request related to a problem? Please describe.**
Right now when you receive an install link and you don't have the app installed, it will open in the browser then immediately redirect to the app store where the app can be installed. 

However after installing the app the information from the install link is forgotten, so the user has to be reminded to go back to the app where they received the link and press it again. 

That's a bit cumbersome.

**Describe the solution you'd like**
The proposed solution instead works in a way that when you open the invite link, it takes you to a page where you will have to press a button (that in the background copies the link to the clipboard).

Then it goes to the app store page of the app and then when the app is running the first time we can look for an invite link on the clipboard.

I think it’s even better this way because on the page where they have to press a button, we can actually explain what’s going on and what they are supposed to do.

The case where it’s not working is that when the user does not finish installing the app or if it takes long and the user switches to an other app that changes the clipboard, but this should be an edge case anyhow

**Describe alternatives you've considered**

1. Using browser fingerprinting

This technique doesn't require user interaction on the website, but instead it generates a unique ID based on the properties of the user's browser and use it as a key to save the link information in a backend service. Then after install it generates the same ID and looks up the backend service for the link information.

This is quite fishy, because it depends on browser fingerprinting which may not be 100% reliable and is prone to failure when the user does not open the app after installing, but later instead. Also there are commercial companies providing products that supports this, but incidentally the two I found is blocked by the adblocker that I use:
- https://branch.io
- https://appflyer.com

2. Using cookies or localstorage

This is similar to the previous, but instead of using browser fingerprinting it uses standard browser APIs with cookies or localstorage. With that it's not necessary to share data with a backend service, so it's better from a tracking standpoint however it's not working with private browsing mode (which may be not an issue) and more importantly, it's not working on iOS 12+ (according to my limited research) which defeats the point.

**Additional context**
These techniques are called [deferred deep linking](https://en.wikipedia.org/wiki/Deferred_deep_linking) and on Android it's natively supported, so the clipboard trick may not be necessary (but maybe it's better not trust Google with that data either).

**Comment by vujevits** [comment]
Yes, this would be very useful! There are some minor ux/design questions regarding how and when display the deep-linked content after installing the application and starting it the first time. Probably it could have a slightly different onboarding flow from the simple fresh install.
